images/linux/scripts/installers/aws-sm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+################################################################################
+##  File:  aws-sm.sh
+##  Desc:  Installs AWS CLI Session Manager
+##         Requires Python >=3.6, must be run as non-root user after toolset installation
+################################################################################
+
+# Download and install aws cli session manager
+curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"
+sudo dpkg -i session-manager-plugin.deb
+
+sessionPlugin=$(session-manager-plugin)
+if [[ $sessionPlugin == *"was installed successfully"* ]]
+then
+    echo "$sessionPlugin"
+    exit 0
+else
+    echo "$sessionPlugin"
+    exit 1
+fi
```

Comment:
Please update documentation about version of session-manager-plugin

Reply:
Added line for documentation: https://github.com/actions/virtual-environments/pull/1169/files#diff-0ebbf504ad9890a7a33b9ae7194239d3R52